### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by seanim0920**
## Problem
PR quality should be improved.

## Proposed Solution
Add a PR template to encourage explaining the purpose and details of the PR. This should ensure every PR is meaningful.

## Steps
* Added pull_request_template.md file to .github folder

## Results
Opening a PR should pre-populate the form with the template.

## Warnings
Needs to be merged in to test.

TASK_pyLuAoOS

### CODE REVIEW

**Comment by tifapp** [comment]
Linked Ticket: https://trello.com/c/pyLuAoOS

**Review by mhayes853 — DISMISSED** [review]
If it works for you, then so be it. But for simple PRs, you may not want to enforce this as it could easily cause clutter.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Describe the deliverable/results of your changes. May include screenshots, request/response snippets, or an example workflow.
+
+## Warnings
+Highlight any changes that might affect the experience of other developers or end-users.
+
+## Other Changes
+Detail any additional changes included in this pull request that do not fall into the above categories.
+
+## Follow Ups
+List any remaining work that is not completed in this pull request:
```
Comment (seanim0920) [inline .github/pull_request_template.md]: could specify that ticket links should be included

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Pull Request Template
+
+## Problem/Purpose
```
Comment (seanim0920) [inline .github/pull_request_template.md:4]: emphasize importance of first section

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Describe the problems or issues that this pull request aims to address.
+
+## Proposed Solution
+Detail the approach taken to solve the problems listed above from a high level.
+
+## Steps
```
Comment (seanim0920) [inline .github/pull_request_template.md]: mention sections that may not be as relevant

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# Pull Request Template
```
Comment (seanim0920) [inline .github/pull_request_template.md:1]: mention that the format is free to modify later

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+## Problem/Purpose
+Describe the problems or issues that this pull request aims to address.
+
+## Proposed Solution
```
Comment (seanim0920) [inline .github/pull_request_template.md:7]: solution and thought process

**Review by suryatiru — APPROVED** [review]

**Pull request merged**